### DISCUSSION TITLE
Recordset pagination in a view/infolist

### DISCUSSION THREAD

**Discussion by dagalumin** [filamentphp/filament#12865] [Help]
### Package

Form builder

### Package Version

v3.2.76

### How can we help you?

We know that the basic behavior for a Resource in Filament is to have a Table listing records, then you click on "View" or "Edit" on one record, and this opens the View/Infolist or Form for that single record. Then you have to dismiss it and go back to the Table if you want to see another record.

What I need to achieve is to have the possibility to paginate the records shown into a View/Infolist: so that if we have a view/infolist showing one record with all its fields, you can go/paginate to other records just pushing a "Next" or "Previous" button, thus showing the View of the next/previous record on the recordset - without coming back to the Table. The recordset could be the result of a query.

I would be very grateful if someone can point me out in the right strategy to achieve this way to navigate/paginate between records of a Resource in a View/Infolist, so not using a Table.
Thank you for any hint.

**Comment by renatofrota** [comment]
[Navigating Filament Pages with Previous and Next Buttons](https://filamentphp.com/content/leandrocfe-navigating-filament-pages-with-previous-and-next-buttons)
- by @leandrocfe 🥇 Filament contributor
